src/CommandingOfficers/Ave.java

```diff
@@ -46,8 +46,8 @@
  *    Ave's units gain a 10 percent increase in firepower.
  *
  * Glacio:
- *    Increases the snow-aura around her buildings by 2 spaces for the next turn.
- *    Snows on every tile in a 2-space radius around each of her units.
+ *    Increases the snow-aura around her buildings by 3 spaces for the next turn.
+ *    Snows on every tile in a 3-space radius around each of her units.
```

Comment:
You appear to have missed updating the code for this.

Reply:
yarr